platforms/documentation/docs/src/docs/userguide/glossary.adoc

```diff
@@ -50,6 +50,65 @@ For more information, see the sections on <<dependency_configurations.adoc#sub:w
 +
 NOTE: The word "configuration" is an overloaded term that has a different meaning outside of dependency management.
 
+[[sub:cross_configuration]]
+Cross-Configuration ::
+See <<sub:cross_project_configuration,Cross-Project Configuration>>.
+
+[[sub:cross_project_configuration]]
+Cross-Project Configuration ::
+Cross-project configuration refers to managing and customizing multiple subprojects within a multi-project build.
++
+It allows you to define common settings, dependencies, and tasks in a shared `build.gradle(.kts)` or `settings.gradle.(kts)` file, typically located in the root project:
++
+[source,groovy]
+.build.gradle
+----
+subprojects {
+    apply plugin: 'java'
+    repositories {
+        mavenCentral()
+    }
+    dependencies {
+        testImplementation 'junit:junit:4.13.2'
+    }
+}
+----
++
+Cross-project configuration usually breaks `Project Isolation` and `Parallel Projects` so you should use convention plugins or the proper APIs whenever possible:
++
+[source,groovy]
+.settings.gradle
+----
+gradle.lifecycle.beforeProject {
+    repositories {
+        mavenCentral()
+    }
+}
+----
++
+The following example of cross-project configuration should *always* be avoided:
++
+[source,groovy]
+.subprojectA/build.gradle
+----
+tasks.register("customTask") {
+    // Avoid this! Directly referencing a task or property from another subproject
+    dependsOn project(":subprojectB").tasks.named("someTask")
```

Comment:
🤔 It's true we should avoid this.  However, it's also true that using `dependsOn` for anything but a lifecycle task (a task that does no work) isn't recommended either.

I think this would be a better example if `customTask` used an output of `someTask` as an one of its inputs.  This would make the snippet longer, but would be a more realistic example.  I don't think many people would expect to use `dependsOn` between projects like this.

Reply:
Fixed